catalyst/utils/torch.py

```diff
@@ -357,7 +357,7 @@ def normalize(samples: Tensor) -> Tensor:
     Returns:
         normalized tensor with the same shape
     """
-    norms = torch.norm(samples, p=2, dim=1).unsqueeze(1)
```

Comment:
why? you could `normalise(features.detach())`

Reply:
what exact `normalise` are you talking about?

Reply:
reworked